webapp/src/monaco.tsx

```diff
@@ -624,26 +629,43 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     display(): JSX.Element {
         const showErrorList = pxt.appTarget.appTheme.errorList;
         const isAndroid = pxt.BrowserUtils.isAndroid();
+        const isWinApp = pxt.BrowserUtils.isWinRT();
         return (
             <div id="monacoEditorArea" className={`monacoEditorArea ${isAndroid ? "android" : ""}`} style={{ direction: 'ltr' }}>
                 {this.isVisible && <div className={`monacoToolboxDiv ${(this.toolbox && !this.toolbox.state.visible && !this.isDebugging()) ? 'invisible' : ''}`}>
                     <toolbox.Toolbox ref={this.handleToolboxRef} editorname="monaco" parent={this} />
                     <div id="monacoDebuggerToolbox"></div>
                 </div>}
-                <div id="monacoEditorRightArea" className="monacoEditorRightArea">
-                    <div id='monacoEditorInner'>
-                        <MonacoFlyout ref={this.handleFlyoutRef} fileType={this.fileType}
-                            blockIdMap={this.blockIdMap}
-                            moveFocusToParent={this.moveFocusToToolbox}
-                            insertSnippet={this.insertSnippet}
-                            setInsertionSnippet={this.setInsertionSnippet}
-                            parent={this.parent} />
+
+                { isWinApp ?
+                    <div id="winAppError">
+                        <img className="ui medium centered image" alt={lf("An image of a shrugging board")} src={pxt.appTarget.appTheme.winAppDeprImage}/>
```

Comment:
The full text is "Oops! Text editing is only available on the MakeCode website. Go here for more information

Reply:
wasn't sure if there was an easier way to embed links in a sentence

Reply:
yeah hmm i don't know if this is better, but you could do something like 

```
const msg = lf("Oops! etc {0}MakeCode website{0}. Go {0}here{0} for more..")
const parts = msg.split("{0}")
```

and then use `parts[0]`, `parts[1]` or whatever in your dialog... maybe a better unique separator than "{0}". this WOULD be a little easier to localize, i think

Reply:
right the less dumb way is 

```
const msg = lf("Oops! ... available on the {0}. Go {1} for more..")
```

then split on `{\d}` and replace the {0} with `lf("MakeCode website") in a link, etc

Reply:
ohoooo

Reply:
Yeah, should be what shannon says here ^^ -- as written the words won't be ordered correctly in other languages.